### ISSUE TITLE
What is the maximum number of iterations?

### ISSUE THREAD

**Issue by taoyunuo** [vanhuyz/CycleGAN-TensorFlow#47]
I can not understand how to set the max-iterations in the code, Please help me to understand,thanks

**Comment by vanhuyz** [comment]
There is no max-iterations in the code. Your must stop the training progress by hand by pressing Ctrl+C. Please check out this issue https://github.com/vanhuyz/CycleGAN-TensorFlow/issues/45.

Maybe I should have included this in README :)

**Comment by u10116032** [comment]
```javascript
      learning_rate = (
          tf.where(
                  tf.greater_equal(global_step, start_decay_step),
                  tf.train.polynomial_decay(starter_learning_rate, global_step-start_decay_step,
                                            decay_steps, end_learning_rate,
                                            power=1.0),
                  starter_learning_rate
          )
      )
```
According to the code above in model.py,
if the global_step get around 200000, the learning rate will decay to 0, which means the model could never change right?

**Comment by vanhuyz** [comment]
Yes! 👍
